gen/src/write.rs

```diff
@@ -936,7 +936,10 @@ fn write_type(out: &mut OutFile, ty: &Type) {
 
 fn write_alias(out: &mut OutFile, alias: &TypeAlias) {
     if let Some(namespace) = &alias.namespace {
-        let path = namespace.path_for_type(&alias.ident);
+        // Review TODO: Is this unwrap fine? i.e. is it ok to assume that, if
+        // the TypePath parsed, that it has at least one segment?
+        let remote_type = &alias.ty.path.segments.last().unwrap().ident;
```

Comment:
Is this unwrap (and the one below) fine or is there a better way to handle this?

Reply:
Obsolete